ch04.adoc

```diff
@@ -185,97 +187,294 @@ The `computed_standard_name` attribute indicates that the values in variable
-Date/times before (and excluding) 1582-10-5 0:0:0 are in the Julian calendar.
+The Gregorian and Julian calendars have the same lengths of their months; they differ only in respect of the rules that decide which years are leap years.
+In the **`standard`** calendar, date/times after and including 1582-10-15 0:0:0 are in the Gregorian calendar, in which a year is a leap year if either (i) it is divisible by 4 but not by 100 or (ii) it is divisible by 400.
+Date/times before (and excluding) 1582-10-5 0:0:0 are in the Julian calendar, in which any year that is divisible by 4 is a leap year.
 Year 1 AD or CE in the **`standard`** calendar is also year 1 of the **`julian`** calendar.
-In the **`standard`** calendar, 1582-10-15 0:0:0 is exactly 1 day later than 1582-10-4 0:0:0 and the intervening dates are undefined.
-Therefore it is recommended that date/times in the range from (and including) 1582-10-5 0:0:0 until (but excluding) 1582-10-15 0:0:0 should not be used as reference in **`units`**, and that a time coordinate variable should not include any date/times in this range, because their interpretation is unclear.
-It is also recommended that a reference date/time before the discontinuity should not be used for date/times after the discontinuity, and vice-versa.
+Negative years are invalid in time coordinates and reference date/times in the **`standard`** calendar.
+In the **`standard`** calendar, 1582-10-15 0:0:0 is exactly 1 day later than 1582-10-4 0:0:0.
+Therefore date/times in the range from (and including) 1582-10-5 0:0:0 until (but excluding) 1582-10-15 0:0:0 are invalid, and must not be used as reference in **`units`**.
+It is recommended that a reference date/time before the discontinuity should not be used for date/times after the discontinuity, and vice-versa.
+See also <<leap-seconds>>.
 
-**`proleptic_gregorian`**:: A calendar with the Gregorian rules for leap-years extended to dates before 1582-10-15.
+**`proleptic_gregorian`**:: A calendar with the Gregorian rules for leap years extended to dates before 1582-10-15.
 All dates consistent with these rules are allowed, both before and after 1582-10-15 0:0:0.
+See also <<leap-seconds>>.
 
 **`julian`**:: Julian calendar, in which a year is a leap year if it is divisible by 4, even if it is also divisible by 100.
+Year 1 AD or CE in the **`julian`** calendar is also year 1 of the **`standard`** calendar.
+Negative years are invalid in time coordinates and reference date/times in the **`julian`** calendar.
+See also <<leap-seconds>>.
+
+**`utc`**:: A Gregorian calendar __with__ leap seconds as prescribed by UTC.
+Date/times before 1958-01-01 0:0:0 are not allowed in this calendar.
+Date/times in the future are not allowed in this calendar, because it is unknown when future leap seconds will occur.
```

Comment:
I *think* some warning is given before a leap second is introduced -- so we could go a bit in the future (wikipedia says " leap seconds are announced only six months in advance.") -- but I can't find a formal reference for that -- so I guess ruling out the future altogether is probably wise.